CRM/Contact/Form/Contact.php

```diff
@@ -807,7 +807,7 @@ public function buildQuickForm() {
     CRM_Contact_Form_Location::buildQuickForm($this);
 
     // add attachment
-    $this->addField('image_URL', array('maxlength' => '60', 'label' => ts('Browse/Upload Image')));
+    $this->addField('image_URL', array('maxlength' => '255', 'label' => ts('Browse/Upload Image')));
```

Comment:
That is because `image_URL` column stores the absolute path and a 32 characters long has is added to ensure uniqueness.